pkg/build/strategies/sti/sti.go

```diff
@@ -599,12 +607,20 @@ func (builder *STI) Execute(command string, user string, config *api.Config) err
 				continue
 			}
 			// The log level > 3 forces to use glog instead of printing to stdout
-			if glog.V(3) {
+			if glog.Is(3) {
```

Comment:
since "glog" is now the wrapper, i don't think this logic is needed anymore..you can just send everything to the glog wrapper and let it decide what to do with it.
(same for the other if checks in this file)